examples/examples/tower-load.rs

```diff
@@ -318,8 +318,8 @@ async fn load_gen(addr: SocketAddr) -> Result<(), Err> {
         .layer(request_span::layer(req_span))
         .timeout(Duration::from_millis(200))
         .service(Client::new());
-
-    while let Some(_) = time::interval(Duration::from_millis(50)).next().await {
+    let mut interval = time::interval(Duration::from_millis(50));
+    while interval.next().await.is_some() {
```

Comment:
I still like if let some better 🤷

Reply:
yeah, i personally think it was fine, but clippy doesn't agree.